src/test/java/stepdefinitions/APIStepDefinitions.java

```diff
@@ -0,0 +1,68 @@
+package stepdefinitions;
+
+
+import io.cucumber.java.en.*;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import pages.ContactUsPage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.restassured.RestAssured.*;
+
+public class APIStepDefinitions {
+
+    private WebDriver driver;
+    private Response response;
+
+    public APIStepDefinitions(){this.driver=DriverManager.driver;}
+
+
+    @When("I create get request from {string}")
+    public void i_create_get_request_from(String url) {
+        response= given().accept(ContentType.JSON).
+                           and().
+                           when().
+                           get(url);
+
+    }
+    @And("Verify the status code is {int}")
+    public void verify_the_status_code_is(Integer statusCode) {
+        Integer actualStatusCode=response.getStatusCode();
+        Assert.assertEquals(statusCode,actualStatusCode);
+
+    }
+
+    @And("Verify the content type is {string}")
+    public void verifyTheContentTypeIs(String contentType) {
+        String actualContentType=response.contentType();
+        Assert.assertEquals(contentType,actualContentType);
+    }
+
+    @Then("Fill the contact form with the data comes from given API")
+    public void fill_the_contact_form_with_the_data_comes_from_given_api() {
```

Comment:
With the way we are using this REST call: just to get some test data, a suggestion would be to move this code out of the step definitions into a different class to reduce the amount of cucumber steps needed for test scenario.